app/assets/javascripts/application.js

```diff
@@ -34,4 +34,6 @@
 //= require modernizr
 
 //= require orangelight
+//= require availability
```

Comment:
Is the explicit requires just a matter of preference? The require tree statement is enough for the asset pipeline.